_dasldemos/proof-how-to.md

```diff
@@ -342,16 +305,14 @@ If you find something is not working with the app or you find a bug, please help
 - [Developing WDL Workflows](https://hutchdatascience.org/Developing_WDL_Workflows/)
 - [Open WDL Documentation](https://docs.openwdl.org/en/latest/)
 
-#### PROOF mechanics
-- [R Client for the PROOF-API](https://github.com/getwilds/proofr)
-- [Shiny Cromwell](https://github.com/getwilds/shiny-cromwell)
-- [rcromwell](https://github.com/getwilds/rcromwell)
+#### PROOF technical documentation and repositories [here](https://sciwiki.fredhutch.org/datascience/proof/#technical-documentation-and-repositories)
 - [PROOF Troubleshooting](/dasldemos/proof-troubleshooting)
 
 
 ## Document Contributions
 We would love to get feedback on this document from the community and would love to hear how we can make improvements to make this more helpful to you! Feel free to [email](mailto:wilds@fredhutch.org) us at or directly make recommendations on [GitHub](https://github.com/FredHutch/wiki/).
 
+
```

Comment:
Prior to approve you must remove the author list from this page.  It is not consistent with the norms of this or any wiki.  Responsibility and contributions to pages is documented through Github commit history and that is all.

Reply:
@vortexing -- author list has been removed from both articles, thanks for catching that!